ui/src/app/app-routing.module.ts

```diff
@@ -209,37 +205,20 @@ const routes: Routes = [
                             children: [
                               {
                                 path: '',
-                                component: CohortReviewComponent,
-                                data: {
-                                  title: 'Review Cohort Participants',
-                                  breadcrumb: BreadcrumbType.Cohort,
-                                  helpContentKey: 'reviewParticipants'
-                                }
+                                component: AppRouting,
+                                data: {},
                               }, {
                                 path: 'participants',
-                                component: TablePage,
-                                data: {
-                                  title: 'Review Cohort Participants',
-                                  breadcrumb: BreadcrumbType.Cohort,
-                                  helpContentKey: 'reviewParticipants'
-                                }
+                                component: AppRouting,
+                                data: {},
                               }, {
                                 path: 'cohort-description',
-                                component: QueryReportComponent,
-                                data: {
-                                  title: 'Review Cohort Description',
-                                  breadcrumb: BreadcrumbType.Cohort,
-                                  helpContentKey: 'cohortDescription'
-                                }
+                                component: AppRouting,
+                                data: {},
                               }, {
                                 path: 'participants/:pid',
-                                component: DetailPageComponent,
-                                data: {
-                                  title: 'Participant Detail',
-                                  breadcrumb: BreadcrumbType.Participant,
-                                  shouldReuse: true,
-                                  helpContentKey: 'reviewParticipantDetail'
-                                }
+                                component: AppRouting,
+                                data: {shouldReuse: true},
```

Comment:
@als364 @petesantos Tagged both of you for review to get your input on this.

I left `shouldReuse: true` here to prevent `DetailPage` component from unmounting and remounting every time the `:pid` param changes. Do you know of a way to do this in the new router?

Reply:
Are you seeing the component unmounting and remounting? I wouldn't think it would unmount and remount, just receive new props.

Reply:
Yeah, definitely unmounts and mounts again. If I put console.logs in `componentDidMount` and `componentWillUnmount` they print every time I change participants. It also changes back to the default Summary tab when we want it to stay on the current tab, Conditions in this example.

Without `{shouldReuse: true}`:

https://user-images.githubusercontent.com/40036095/112363059-197b4900-8ca3-11eb-815e-588233b0d159.mov


With `{shouldReuse: true}`:

https://user-images.githubusercontent.com/40036095/112363127-2861fb80-8ca3-11eb-9d93-dccb9401f94e.mov

Reply:
Hm. It's kind of weird that the angular and react routers interact in this way.

This [React router documentation](https://reactrouter.com/web/api/Route/render) suggests that by using `render` instead of `component` in the router, this unmounting/remounting can be avoided. Could you try that?

Reply:
Yeah, I saw that as well, but looks like we're already using `render`, unless I'm looking in the wrong place: https://github.com/all-of-us/workbench/blob/master/ui/src/app/components/app-router.tsx#L46

Reply:
Oh, I missed that! Seems like this is just a weird interaction between the Angular and React router then. I'll make a note in the ticket to delete the angular router to make sure that this doesn't happen when the angular router is deleted.

Reply:
Sounds good, thanks!

Reply:
Ended up removing `shouldReuse: true` again. It was causing `helpContentKey` not to get set when navigating between participants, which caused the right sidebar to disappear. Staying on the same tab between participants is not super critical anyway, can wait till the Angular router goes away.

Reply:
Ah, interesting. Would you mind making a bug ticket under [this epic](https://precisionmedicineinitiative.atlassian.net/browse/RW-2077) to follow this up, blocked by [this ticket](https://precisionmedicineinitiative.atlassian.net/browse/RW-6330)?

Reply:
Done: https://precisionmedicineinitiative.atlassian.net/browse/RW-6527